core/message_handler.go

```diff
@@ -12,8 +12,9 @@ import (
 // Ensure that App implements p2p.MessageHandler.
 var _ p2p.MessageHandler = &App{}
 
-type MessageHandler struct {
+type OrderSelector struct {
```

Comment:
`OrderSelector` should probably be unexported. (`MessageHandler` should have been too; I just didn't notice it at the time).